aten/CMakeLists.txt

```diff
@@ -32,6 +32,17 @@ ENDIF(NOT MSVC)
 
 INCLUDE(CheckCXXSourceCompiles)
 
+# windef.h will define max/min macros if NOMINMAX is not defined
+IF(MSVC)
+  add_definitions(/DNOMINMAX)
+ENDIF(MSVC)
+
+# # Pre 2015 MSVC can't deal with varible length arrays
```

Comment:
I don't think we support pre-2015 MSVC, not that we use VLA.

Reply:
I'd prefer to leave this, because it bit me in the ass.

Reply:
Uncomment it? I don't see how having this commented out in the CMakeList is helping anyone.